contracts/modules/book-room/TestLLrevert.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
```

Comment:
Could u move this file to ./test/sol directory?
And to avoid executing those test case like, testDupAddRevert(),  testRemoveNoExistRevert(), testRemoveWrongPreRevert(), try remove/alter the prefix `test` (case-sensitive).